test/index.js

```diff
@@ -17,9 +17,15 @@ var testMap = [
         ieCompat: true
     }],
     [{
-        strictMath: true,
+        math: 'strict-legacy',
         ieCompat: true
-    }, 'strict-math/'],
+    }, 'math/strict-legacy/'],
+    [{
+        math: 'parens-all'
+    }, 'math/parens-all/'],
+    [{
+        math: 'parens-division'
+    }, 'math/parens-division/'],
     [{strictMath: true, strictUnits: true, javascriptEnabled: true}, 'errors/',
```

Comment:
Should the `strictMath` option be updated here (and in other lines below) as well?  Or will it not matter?

Reply:
I left some in to verify that setting the legacy option worked as expected. I should make a note of it though.